lms/static/sass/base/_layouts.scss

```diff
@@ -3,7 +3,8 @@
 
 // overriding existing styles on the body element
 // .view-incourse scopes these rules to be specific to student being in a course
-body.view-incourse {
+body.view-incourse,
+body.is-in-course {
```

Comment:
Ah ok. The notes case makes sense, but I'm keeping that naming cleanup out of this PR. A quick check of the team's partial led me to expect .view-teams for that case however. I'll double check locally for me prior to thumbs just as a double check. Thanks for clarifying!